examples/wordcloud_cn.py

```diff
@@ -12,17 +12,18 @@
 at the same time using wordcloud with jieba very convenient
 """
 
+import os
+
 import jieba
 jieba.enable_parallel(4)
 # Setting up parallel processes :4 ,but unable to run on Windows
-from os import path
 from scipy.misc import imread
 import matplotlib.pyplot as plt
 # jieba.load_userdict("txt\userdict.txt")
 # add userdict by load_userdict()
 from wordcloud import WordCloud, ImageColorGenerator
 
-d = path.dirname(__file__)
```

Comment:
Is that not equivalent but just longer?

Reply:
In my Windows and Linux, 'path.dirname(__file__)' does't work.

``` python
# -*- coding: utf-8 -*-
#  test.py
import os

d = os.path.dirname(__file__)
print(d)
```
touch a file named test.py in your home directory, '/home/yourhome'.
if you run the python file in your home directory, like 'python test.py', print nothing
if you run the python file ,like 'python /home/yourhome/test.py', print '/home/yourhome'

the return value is not certain, which is confusing. 'os.path.split(os.path.realpath(__file__))[0]' always is fine.